lib/utils/moduleAlias.js

```diff
@@ -0,0 +1,38 @@
+// Require our actual `package.json` data for use by this file.
+//
+// NB: `package` is a reserved word
+const packageData = require('../../package.json')
+
+// Extracts and returns the `module-alias` configuration from our `package.json` data.
```

Comment:
Love the documentation, but we should use the docblock format that we've been using elsewhere.  I believe that adheres to [this standard](https://jsdoc.app/).

(side note: I am using [DocBlockr](https://packagecontrol.io/packages/DocBlockr) for sublime, idk if there is a similar one for your IDE)

Key elements are using /** and */ as well as the use of things like `@returns` etc.